src/xdocs/google_style.xml

```diff
@@ -462,7 +462,7 @@
                             <td><a name="4.1.2"/><a href="#4.1.2"><img src="images/anchor.png" alt=""/></a></td>
                             <td>
                                 <a
-                                    href="http://checkstyle.sourceforge.net/reports/google-java-style.html#s4.1.2-blocks-k-r-style">4.1.2 Nonempty blocks: K &amp; R style</a>
+                                    href="http://checkstyle.sourceforge.net/reports/google-java-style-20160712.html#s4.1.2-blocks-k-r-style">4.1.2 Nonempty blocks: K &amp; R style</a>
                             </td>
```

Comment:
I created an example in #3546.
Do we want another issue to make Left and Right curly accept the same tokens? Unless they are equal, or you can explain why they shouldn't be 100% equal, we have areas were one validates a specific token and the other one doesn't.